tarifierung-api.yaml

```diff
@@ -572,17 +572,20 @@ components:
     Error:
       description: Fehlermeldung
       properties:
+        message:
+          type: string
         id:
-          description: Identifikation der Fehlertypes.
           type: string
-        messageId:
-          description: Identifikation der Fehlermeldung.
+        key:
           type: string
-        messages:
-          description: Fehlermeldung
+        args:
           type: array
           items:
-            $ref: "#/components/schemas/LocalizedValue"
+            type: object
```

Comment:
Weiss nicht ob `object` hier eine gute Idee ist. Wie würde der ERP-Hersteller damit umgehen, wenn das ein LocalDate, LocalDateTime, BigDecimal, String, int, etc. in der Liste vorkommen. Er müsste je eigentlich durch probieren, wo ist das parsable?!

Reply:
Sehr guter Input -> ich werde es auf type; string ändern